### PR TITLE
Add fxupdate.csx script for updating versions of dependencies

### PR DESCRIPTION
**Pull request by tmat**

### CODE REVIEW

**Comment by tmat** [comment]
FYI @agocke @cston @dotnet/roslyn-infrastructure 
@MattGertz Ask mode PR - infrastructure only change

**Comment by ManishJayaswal** [comment]
@MattGertz This will help us in responding to new packages from CoreFX, which is happening pretty frequently now.

File: build/scripts/fxupdate.csx [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+
+string UpdateVersionElement(string text, string elementName, string newValue)
+{
+    return Regex.Replace(
+        text,
+        $"<{elementName}>[^<]+</{elementName}>",
+        $"<{elementName}>{newValue}</{elementName}>");
+}
+
+UpdateTargetsFile(Path.Combine(roslynRoot, "build", "Targets", "VSL.Versions.targets"));
```
Comment (agocke) [inline build/scripts/fxupdate.csx]: So are we storing the corefx versions in `VSL.Versions.targets` now?

Comment (tmat) [inline build/scripts/fxupdate.csx]: Yes

**Comment by agocke** [comment]
:+1:

**Pull request merged**